polaris-react/src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.scss

```diff
@@ -22,12 +22,18 @@ $contextual-save-height: 44px;
     --p-color-bg-hover: var(--p-color-bg-hover);
     --p-color-bg-subdued-active: var(--p-color-bg-subdued-active);
     /* stylelint-enable */
-    border-radius: var(--p-border-radius-3);
-    margin: var(--p-space-2);
+    margin-top: 0;
+    margin-bottom: 0;
     height: $contextual-save-height;
     background-color: var(--p-color-bg-input);
     color: var(--p-color-bg-inverse);
     box-shadow: var(--p-shadow-md);
+
+    @media #{$p-breakpoints-md-up} {
```

Comment:
I think this should be using the `#{$p-breakpoints-sm-up}` media query.

https://github.com/Shopify/polaris/assets/11774595/ccec6607-66e6-4cf1-a8e8-e79b2a3b6c95

Reply:
You are correct, will update